pandas/core/computation/pytables.py

```diff
@@ -13,18 +13,18 @@
 
 import pandas as pd
 import pandas.core.common as com
-from pandas.core.computation import expr, ops
+from pandas.core.computation import expr, ops, scope as _scope
 from pandas.core.computation.common import _ensure_decoded
 from pandas.core.computation.expr import BaseExprVisitor
 from pandas.core.computation.ops import UndefinedVariableError, is_term
 
 from pandas.io.formats.printing import pprint_thing, pprint_thing_encoded
 
 
-class Scope(expr.Scope):
+class Scope(_scope.Scope):
     __slots__ = ("queryables",)
```

Comment:
the medium-term goal that motivates this and the other pytables PR is that I _think_ we can type `queryables` as `Dict[str, Any]`, but it'll take a few steps to get there